lib/index.js

```diff
@@ -32,15 +32,22 @@ function fluentAppender(config, layout) {
   const logSender = FluentSender.createFluentSender(tag_prefix, options);
 
   const appender = function(loggingEvent) {
-    const data =  util.format.apply(null, loggingEvent.data);
-    const rec = {
+    const data =  loggingEvent.data[0];
+    const extra = loggingEvent.data.splice(1);
+
+    let rec = {
       timestamp: loggingEvent.startTime.getTime(),
       category: loggingEvent.categoryName,
       levelInt: loggingEvent.level.level,
       levelStr: loggingEvent.level.levelStr,
       context: loggingEvent.context,
       data: data
     };
+
+    if (extra.length) {
+      rec['extra'] = Object.assign(...extra);
```

Comment:
Don't use spread operator.
I want to support Node.js 4 until its [EOL (2018-04-30)](https://github.com/nodejs/Release).